### PR TITLE
DGL-KE Wikimedia Notebook Example

### PR DESCRIPTION
**Pull request by phiamazon**
*Issue #, if available:*

*Description of changes:*
Adding a PR for new example for DGL-KE

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: sagemaker-python-sdk/dgl_kge/kge_wikimedia.ipynb [hunk 1]
```diff
@@ -0,0 +1,1396 @@
...
+            "  File \"/usr/lib/python3.6/dist-packages/pkg_resources/__init__.py\", line 2854, in load_entry_point\n",
+            "    return ep.load()\n",
+            "  File \"/usr/lib/python3.6/dist-packages/pkg_resources/__init__.py\", line 2445, in load\n",
+            "    return self.resolve()\n",
+            "  File \"/usr/lib/python3.6/dist-packages/pkg_resources/__init__.py\", line 2451, in resolve\n",
+            "    module = __import__(self.module_name, fromlist=['__name__'], level=0)\n",
+            "  File \"/usr/local/lib/python3.6/site-packages/dglke-0.1.0.dev0-py3.6.egg/dglke/infer_score.py\", line 25, in <module>\n",
+            "  File \"/usr/local/lib/python3.6/site-packages/dglke-0.1.0.dev0-py3.6.egg/dglke/models/__init__.py\", line 20, in <module>\n",
+            "  File \"/usr/local/lib/python3.6/site-packages/dglke-0.1.0.dev0-py3.6.egg/dglke/models/general_models.py\", line 30, in <module>\n",
+            "ModuleNotFoundError: No module named 'dgl'\n"
```
Comment (aaronmarkham) [inline sagemaker-python-sdk/dgl_kge/kge_wikimedia.ipynb:1357]: Why do you think it breaks down here when it worked earlier?

Comment (phiamazon) [reply sagemaker-python-sdk/dgl_kge/kge_wikimedia.ipynb:1357]: The inference / score function was added to the dgl-ke pip package was added later on. I had python package dependencies challenges. I am re-running now the whole notebook from a clean environment and I will commit a new version.

Comment (ngluna) [reply sagemaker-python-sdk/dgl_kge/kge_wikimedia.ipynb:1357]: @phiamazon Any status update? Did the notebook break after you re-ran the notebook?

**Comment by phiamazon** [comment]
Sorry for the delayed reply. We decided to push it in a different repo. I will go ahead and close this PR.

**Comment by phiamazon** [comment]
We ended up pushing in a different repo.

**Pull request closed without merging**